.github/workflows/sv_pipeline_docker.yml

```diff
@@ -205,7 +132,7 @@ jobs:
       - name: Run build_docker.py
         run: |
           cd ./scripts/docker/
-          python build_docker.py --targets ${{ needs.build_args_job.outputs.target_images }} --image-tag ${{ needs.build_args_job.outputs.images_tag }}
+          python build_docker.py --base-git-commit ${{ needs.build_args_job.outputs.base_sha }} --current-git-commit ${{ needs.build_args_job.outputs.head_sha }} --image-tag ${{ needs.build_args_job.outputs.images_tag }}
```

Comment:
There's also no need to compute the --image-tag. build_docker.py has a default which will work.

Reply:
Updated accordingly.